src/megaclient.cpp

```diff
@@ -6728,7 +6728,7 @@ void MegaClient::mapuser(handle uh, const char* email)
         u = &users[hit->second];
 
         um_map::iterator mit = umindex.find(nuid);
-        if (mit != umindex.end() && mit->second != hit->second && users[mit->second].show != INACTIVE)
+        if (mit != umindex.end() && mit->second != hit->second && (users[mit->second].show != INACTIVE || users[mit->second].userhandle == me))
```

Comment:
Shouldn't this be +        if (mit != umindex.end() && mit->second != hit->second && (users[mit->second].show != INACTIVE **&&** users[mit->second].userhandle **!=** me))